src/rgw/rgw_rest_user.cc

```diff
@@ -981,7 +981,7 @@ void RGWOp_Quota_Set::execute(optional_yield y)
   RGWUser user;
   op_ret = user.init(s, store, op_state, y);
   if (op_ret < 0) {
-    ldout(store->ctx(), 20) << "failed initializing user info: " << op_ret << dendl;
+    ldpp_dout(s, 20) << "failed initializing user info: " << op_ret << dendl;
```

Comment:
you might want to use `this` here instead of `s`.

Reply:
Okay.